dev/oci_env_integration/oci_env_configs/community.compose.env

```diff
@@ -12,8 +12,8 @@ DJANGO_SUPERUSER_USERNAME=admin
 DJANGO_SUPERUSER_PASSWORD=admin
 
 ## Enable github social auth by setting these two values
-SOCIAL_AUTH_GITHUB_KEY='abcd1234'
-SOCIAL_AUTH_GITHUB_SECRET='abcd1234'
+#SOCIAL_AUTH_GITHUB_KEY='abcd1234'
+#SOCIAL_AUTH_GITHUB_SECRET='abcd1234'
```

Comment:
Should these lines be removed to reduce confusion?

Reply:
tbh, i'm not sure why we have these env files here and also profile specific files in the profiles dir.